Sprint-2/2-mandatory-debug/0.js

```diff
@@ -9,6 +10,14 @@ function multiply(a, b) {
 console.log(`The result of multiplying 10 and 32 is ${multiply(10, 32)}`);
 
 // =============> write your explanation here
+// Inside function run the console.log(a * b) is executed, but this function doesn't return anything.
+
+// When using ${multiply(10, 32)}, the function runs and prints 320, but since there’s no return, the value is undefined.
 
```

Comment:
Well done, It is really nice to see you understand what happens if you don't return a value  in a function. if you  just console.log the value, your function return void in the end